x-pack/plugins/alerting/server/rule_type_registry.ts

```diff
@@ -410,6 +412,21 @@ export class RuleTypeRegistry {
   public getAllTypes(): string[] {
     return [...this.ruleTypes.keys()];
   }
+
+  public getIndicesAlias(rulesTypes: string[], spaceId?: string): string[] {
```

Comment:
**question:** can we move this outside the rule type registry? It doesn't seem related to the registry pattern.